roles/openshift_hosted/tasks/router.yml

```diff
@@ -26,6 +26,7 @@
     backup: True
     dest: "/etc/origin/master/{{ item | basename }}"
     src: "{{ item }}"
+    remote_src: "{{ openshift_hosted_router_certificate_from_remote | default(no) }}"
```

Comment:
Set default value in defaults/main.yml and use false to be consistent with other bools